background-change.js

```diff
@@ -0,0 +1,25 @@
+// Javascript for changing the background on page load/reload.
+
+window.addEventListener('load', function() {
+    // Array of background images in the images folder
+    const backgroundImageNames = [
+        'image1.webp',
```

Comment:
This all looks great, the only thing I need changed is swapping the image names from `image1.webp` to`bg1.webp`, that way it's a little more obvious to someone outside the repo. Other than that looks great man!